egui/src/response.rs

```diff
@@ -185,6 +185,17 @@ impl Response {
         self.hovered
     }
 
+    /// The pointer is hovering above this widget or the widget was clicked/tapped this frame.
+    /// this can help to deal with child widgets events
+    pub fn hovered_anyway(&self) -> bool {
+        let pointer = &self.ctx.input().pointer;
+        if let Some(pos) = pointer.interact_pos() {
+            self.rect.contains(pos)
```

Comment:
this will return true even if another egui window is covering your widget. is this really intentional?

Reply:
I didn't think so much ....

Reply:
What is your use case? Would `fn hovered_or_clicked(&self) -> bool { self.hovered() || self.clicked() }` handle it?

Reply:
I want to realize that when hovering the current component, it can display the sub-components, such as "delete button"

![image](https://user-images.githubusercontent.com/14890680/142961409-2960ff4a-e38b-4ace-a01a-48023d4e0c99.png)
![image](https://user-images.githubusercontent.com/14890680/142961548-bffd7192-3937-4b86-a8c6-0fb502380d97.png)